server.test.js

```diff
@@ -0,0 +1,58 @@
+const app = require("./server.js");
+const mysql = require("mysql");
+const supertest = require("supertest");
+const dbconfig = {
+  host: "mscsdb.uwstout.edu",
+  user: "academicSelect",
+  password: "mergeSort45!",
+  database: "academicrequirements",
+};
+let connection;
+
+beforeEach(async () => {
+  connection = mysql.createConnection(dbconfig);
+});
+
+afterEach(async () => {
+  connection.end();
+});
+
+it("test /major endpoint", async () => {
```

Comment:
On at least one of these tests, I think it could be beneficial to thoroughly comment what each line is doing. You could include information such as what test is or is not doing.